mender/values.yaml

```diff
@@ -280,6 +280,29 @@ auditlogs:
     type: ClusterIP
     port: 8080
 
+azure_iot_manager:
+  enabled: true
+  automigrate: true
+  replicas: 1
+  resources:
+    limits:
+      cpu: 50m
+      memory: 128M
+    requests:
+      cpu: 50m
+      memory: 128M
+  affinity: {}
+  image:
+    registry: registry.mender.io
+    repository: mendersoftware/azure-iot-manager
+    tag: mender-3.1.0
```

Comment:
Uhm, this won't work until we release... shall we use mender-master in the meanwhile?
Maybe we should replace all the image tags in the master branch of the Helm chart to mender-master?

Reply:
changed to mender-master. I have no problem in using mender-master everywhere.